com.unity.render-pipelines.universal/Runtime/2D/LightUtility.cs

```diff
@@ -208,8 +224,16 @@ public static List<Vector2> GetFalloffShape(Vector3[] shapePath)
             return extrusionDir;
         }
 
-        public static Bounds GenerateShapeMesh(Mesh mesh, Vector3[] shapePath, float falloffDistance)
+        public static Bounds GenerateShapeMesh(Mesh mesh, Vector3[] shapePath, float falloffDistance, LightCachedMeshData cachedMeshData)
         {
+            bool staticUpload = Application.isPlaying;
+#if !UNITY_EDITOR
+            staticUpload = true;
+#endif
+            if (staticUpload && cachedMeshData)
+                if (cachedMeshData.RequiresUpload())
+                    return cachedMeshData.Upload(mesh);
```

Comment:
Same suggestion as above.

Reply:
Fixed in 94f30ac326f1fc161502821c5d49373b19127b16